doc/DATA_MODEL.md

```diff
@@ -72,6 +72,15 @@ They are typically cached for quick loading, and can be inserted anywhere in the
 
 Comments belong to Notes via `nid`, and each have an author via `uid`; primary key `cid`. Maps also have comments via `nid`, and Answers may also have comments, via `aid`.
 
+Node `status` -- a property of `comments`, can be:
+
+* 0: banned
+* 1: normal
+* 3: draft
+* 4: moderated -- i.e. node created by a first-time poster, and has not yet been "approved"
+
+The status for Node is now the same for comment in our Code base except for status = 5
```

Comment:
How about removing this line? @dinaelhanan

Reply:
ok, will do. Thanks!